week2/homework/src/addToDo.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+let toDoList = [];
+function generateToDo(toDo) {
+  try {
+    toDoList = JSON.parse(fs.readFileSync('toDoList.json'));
+  }
+ catch (error) {
+    console.log(`${error}`);
```

Comment:
perhaps you need to move these line from 10 to 15 to the try close. :)
when I have added a new item I got: `Error: ENOENT: no such file or directory, open 'toDoList.json'`